### PR TITLE
amend deprecation policy on stability framework

### PR DESCRIPTION
**Pull request by logicalhan**
Add some information on what is acceptable in terms of graduating metrics.

/assign @dgrisonnet

### CODE REVIEW

File: content/en/docs/reference/using-api/deprecation-policy.md [hunk 1]
```diff
@@ -441,6 +441,12 @@ deprecating and removing a metric are as follows:
    * **STABLE: 3 releases or 9 months (whichever is longer)**
    * **BETA: 1 releases or 4 months (whichever is longer)**
    * **ALPHA: 0 releases**
+   
+It is permissible, in certain cases, for metrics to skip stability levels in the process
```
Comment (dgrisonnet) [inline content/en/docs/reference/using-api/deprecation-policy.md:445]: The doc on the website is specific to the deprecation policies, maybe we should rather move that to https://github.com/kubernetes/community/blob/53ccd27e43731fe76bff3a00c602d53eb5ed55ae/contributors/devel/sig-instrumentation/metric-stability.md and update that doc while we are at it?

**Comment by Rishit-dagli** [comment]
Do we want to try and merge this PR before the code freeze which starts on Monday?

**Comment by sftim** [comment]
@logicalhan, could you respond to https://github.com/kubernetes/website/pull/42217#discussion_r1276231513 ?

**Comment by sftim** [comment]
> Do we want to try and merge this PR before the code freeze which starts on Monday?

This could land after the release (we'll need to revise the base branch to _main_, or maybe cherry-pick and make a new PR).

File: content/en/docs/reference/using-api/deprecation-policy.md (hunk 1 above)
Comment (logicalhan) [reply content/en/docs/reference/using-api/deprecation-policy.md:445]: Let's update it in both spots. I'll open up another PR.

**Comment by divya-mohan0209** [comment]
Hello @kubernetes/sig-instrumentation-approvers please could y'all have a look and advise from a tech POV so that we can ready it for merge?

**Comment by sftim** [comment]
@logicalhan, would you be willing to propose a similar change to _main_? This one didn't make it before the v1.28 release.

**Comment by logicalhan** [comment]
> @logicalhan, would you be willing to propose a similar change to _main_? This one didn't make it before the v1.28 release.

Sure, I can try to get to it tomorrow.

**Pull request merged**